src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -8920,26 +8919,23 @@ internal sealed class CSharpCompilerDiagnosticAnalyzer
             var emitResult2 = compilation2.Emit(peStream: new MemoryStream(), options: new EmitOptions(metadataOnly: true));
             Assert.False(emitResult2.Success);
             emitResult2.Diagnostics.Verify(
-    // error CS7038: Failed to emit module 'Test.dll'.
-    Diagnostic(ErrorCode.ERR_ModuleEmitFailure).WithArguments("Test.dll").WithLocation(1, 1)
-                );
+                // error CS7038: Failed to emit module 'Test.dll': Module has invalid attributes.
+                Diagnostic(ErrorCode.ERR_ModuleEmitFailure).WithArguments("Test.dll", CodeAnalysisResources.ModuleHasInvalidAttributes).WithLocation(1, 1));
```

Comment:
>CodeAnalysisResources.ModuleHasInvalidAttributes [](start = 86, length = 48)

This is likely going to cause a problem for Spanish_Unit_Tests once the strings are localized.

Reply:
> This is likely going to cause a problem for Spanish_Unit_Tests once the strings are localized.

The string should always be pulled from english resources, the test infrastructure always formats arguments this way. 

---
In reply to: [356197287](https://github.com/dotnet/roslyn/pull/40180#discussion_r356197287) [](ancestors = 356197287)